gdpr/aggregated_config.go

```diff
@@ -13,6 +13,7 @@ type TCF2ConfigReader interface {
 	FeatureOneEnforced() bool
 	FeatureOneVendorException(openrtb_ext.BidderName) bool
 	IntegrationEnabled(config.IntegrationType) bool
```

Comment:
Now that we have `ChannelEnabled` we should be able to delete `IntegrationEnabled` from the interface.